packages/firestore/test/unit/specs/existence_filter_spec.test.ts

```diff
@@ -631,4 +631,189 @@ describeSpec('Existence Filters:', [], () => {
         .expectLimboDocs(...docKeys.slice(50))
     );
   });
+
+  specTest('Bloom filter handles new document adds', [], () => {
+    const query1 = query('collection');
+    const docs = [];
+    for (let i = 0; i < 100; i++) {
+      docs.push(doc(`collection/doc${i}`, 1000, { v: 1 }));
+    }
+    const docKeys = docs.map(item => item.key);
+
+    const bloomFilterProto = generateBloomFilterProto({
+      contains: docs.slice(0, 50),
+      notContains: docs.slice(50),
+      bitCount: 1000,
```

Comment:
Is the `bitCount` and `hashCount` relevant for this test case? If no, please don't specify them because they add "noise" to the test.